File: C++/همایش زندگی بهتر.cpp
#include <iostream>
using namespace std;
int main()
{
    bool right{false};
    int column, row;
    cin >> row >> column;
    if (column <= 10)
    {
        right = true;
    }
    else
    {
        column = (20 - column) + 1;
    }
    row = (10 - row) + 1;
    if (right)
    {
        cout << "Right " << row << " " << column;
    }
    else
    {
        cout << "Left " << row << " " << column;
    }
}

File: Python/همایش زندگی بهتر.py
row, column = input().split()
row = int(row)
column = int(column)
if column <= 10:
    right = True
    k = column
else:
    right = False
    k = (20-column)+1
a = (10-row)+1
if right:
    print('Right', a, k)
else:
    print('Left', a, k)
